data/README.md

```diff
@@ -31,15 +31,18 @@ instance, they should be in the same format as the files below.
 ### **all-pseudomonas-gene-normalized.pcl**
   Contains normalized expression levels for every gene in each sample in
   the compendium. In this file, genes are rows, and samples are columns.
-  This is the input expression data for building an ADAGE model.
+  This is the input expression data for building an ADAGE model, and was
+  generated using
+  [this script](https://bitbucket.org/greenelab/eadage/src/tip/data_collection/data_collection.sh).
 
 ### **sample_signature_activity.txt**
   Contains signature (node) activity levels (as absolute values) for each
   sample in the compendium derived from the eADAGE machine learning model.
   In this file, samples are rows, and signatures are columns. This file is
   based on the
   `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt`
-  file, which is now in the `data/old/` folder.
+  file, which is now in the `data/old/` folder, which was generated by
```

Comment:
Maybe combine these two "which" clauses into one like:
```which is now in the `data/old/` folder and was generated by ...```

Reply:
For sure, good call - that is much clearer.